buildSrc/src/main/kotlin/otel.javaagent-testing.gradle.kts

```diff
@@ -1,100 +1,20 @@
-import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
-
 plugins {
   id("net.bytebuddy.byte-buddy")
 
-  id("otel.instrumentation-conventions")
-  id("otel.javaagent-codegen")
-  id("otel.shadow-conventions")
+  id("io.opentelemetry.instrumentation.javaagent-testing")
+
+  id("otel.java-conventions")
 }
 
+evaluationDependsOn(":testing:agent-for-testing")
+
 dependencies {
-  // Integration tests may need to define custom instrumentation modules so we include the standard
-  // instrumentation infrastructure for testing too.
-  compileOnly(project(":instrumentation-api"))
-  compileOnly(project(":javaagent-api"))
-  compileOnly(project(":javaagent-bootstrap"))
-  // Apply common dependencies for instrumentation.
-  compileOnly(project(":javaagent-extension-api")) {
-    // OpenTelemetry SDK is not needed for compilation
-    exclude(group = "io.opentelemetry", module = "opentelemetry-sdk")
-    exclude(group = "io.opentelemetry", module = "opentelemetry-sdk-metrics")
-  }
-  compileOnly(project(":javaagent-tooling")) {
-    // OpenTelemetry SDK is not needed for compilation
-    exclude(group = "io.opentelemetry", module = "opentelemetry-sdk")
-    exclude(group = "io.opentelemetry", module = "opentelemetry-sdk-metrics")
-  }
   compileOnly("net.bytebuddy:byte-buddy")
```

Comment:
Hoping this won't need to be added for external dependencies (wonder if we even need it internally I didn't try deleting it yet)

Reply:
If this (and slf4j-api) are needed, we should probably just make them `api` dependencies on `javaagent-api`

Reply:
I don't think that `testing-common` and `javaagent-api` need byte buddy, it should be an `api` dependency in `javaagent-extension-api`

Reply:
Cool we already had it